docs/07_how-to/09_fdc3.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: 'How to set up interoperability'
+sidebar_label: 'Interoperability/FDC3'
+id: ht-fdc3
+keywords: [fdc3, interoperability]
+tags:
+    - fdc3
+    - interoperability
+
+---
+
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+A Genesis app can use the FDC3 standard for interoperability between front-end applications. 
+
+So, if you have apps that provide services such as charting or news, then you can set up your app so that can share messages with those apps at the front end.
+
+For example, a user could click on a row in a grid of data in your Genesis application and then view view news or a chart on that instrument in a separate app window.
+
+You can achieve this with relatively few lines of code.
+
+The main steps involved in setting this up are:
+
+- Set up a channel
+- Set up events on those channels
+- Set up listeners that can listen to the channels
+
+There are four example apps in total:
+
+- FDC3 blotter
+- intent-listener is an application with a form for submitting trades. 
+- system-channel-listener listens to the active color channel for FX pair events.
+- app-channel-listener listens to the `customAppChannel` for FX trade events.
+
+## How it works
+
+### FDC3 web client
+We have created an example application to demonstrate how you can set up interaction between your Genesis application and other display applications.
+
+The application was generated using Genesis Create and has been adapted to demonstrate the features of `foundation-fdc3` library.
+
+It is designed to serve several 'fdc3-apps' that use the interop features of FDC3 to interact with each other.
+
+Each 'fdc3-app' is a different route on the web application but they are loaded in a different Desktop Agent container. In real life, your apps will have different origins.
+
+## Running in Desktop Agent
+
+The application should be loaded in an [FDC3 Desktop Agent](https://fdc3.finos.org/docs/api/ref/DesktopAgent) to ensure that the FDC3 functionality works.
+
+To run the app, go to the **./fdc3 client** directory and start the **how-touiexamples** server.
+
+```shell
+npm run dev
+```
+
+This starts the server that serves the urls accessed by the apps in the desktop agent. 
+
+### FDC3 blotter
```

Comment:
Again, not sure why we're just copying the README over here. It's going to break over time, can't we just give high level overview (feels like this is lacking) and point to the README for more details? This one doesn't actually even point at the readme or app anywhere it seems.